Resources/Textures/Clothing/OuterClothing/Hardsuits/atmospherics.rsi/meta.json

```
@@ -2,6 +2,7 @@
   "version": 1,
   "license": "CC-BY-SA-3.0",
   "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/97ec5ed1a0fc8263df5df5a5afbb653d4684992e & icon made by github:Morb0, reptilian made by kuro(388673708753027083). Vox state made by Flareguy for SS14",
```

Biome (1.9.4)

[error] The key copyright was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)